### PR TITLE
feat: add inline keyboard for editing memo visibility and pinning

### PR DESCRIPTION
**Pull request by wolfsilver**
image

Fixes https://github.com/usememos/telegram-integration/issues/27

### CODE REVIEW

**Comment by csznet** [comment]
Great feature, I hope to approve it as soon as possible.

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

File: memogram.go [hunk 1]
```diff
@@ -208,6 +211,124 @@ func (s *Service) startHandler(ctx context.Context, b *bot.Bot, m *models.Update
 	})
 }
 
+func (s *Service) keyboard(memo *v1pb.Memo) *models.InlineKeyboardMarkup {
+	// add inline keyboard to edit memo, Public，Private， Pin
```
Comment (boojack) [inline memogram.go:215]:
```suggestion
	// add inline keyboard to edit memo's visibility or pinned status.
```

File: memogram.go [hunk 2]
```diff
@@ -208,6 +211,124 @@ func (s *Service) startHandler(ctx context.Context, b *bot.Bot, m *models.Update
...
+			},
+		}
+}
+
+func (s *Service) callbackQueryHandler(ctx context.Context, b *bot.Bot, update *models.Update) {
+	callbackData := update.CallbackQuery.Data
+	userID := update.CallbackQuery.From.ID
+
+	slog.Info("bot callbackData", slog.String("callbackData", callbackData))
+
```
Comment (boojack) [inline memogram.go:241]:
```suggestion
```

**Comment by wolfsilver** [comment]
@boojack updated.

**Pull request merged**